packngo.go

```diff
@@ -230,6 +231,45 @@ func (c *Client) Do(req *http.Request, v interface{}) (*Response, error) {
 	return &response, err
 }
 
+// dumpDeprecation logs headers defined by
+// https://tools.ietf.org/html/rfc8594
+func dumpDeprecation(resp *http.Response) {
+	deprecation := resp.Header.Get("Deprecation")
+
+	links := resp.Header.Values("Link")
```

Comment:
The server may use a single comma separate Link header instead of multiple, as in https://tools.ietf.org/html/draft-ietf-httpapi-deprecation-header-01#section-10

It's going to be better to handle this as a comma separate list by using `.Get` instead of `.Values`, I believe `.Get` will handle both single comma separated `Link` headers and multiple `Link` headers (which may include comma separated single `Link` headers).